build.gradle.kts

```diff
@@ -169,3 +170,11 @@ tasks {
 
 println("Java version: ${System.getProperty("java.version")}")
 println("Version: $version")
+val compileKotlin: KotlinCompile by tasks
+compileKotlin.kotlinOptions {
+	jvmTarget = "1.8"
```

Comment:
tasks.with<KotlinCompile>...
`JavaVersion.VERSION_1_8.toString()`

Reply:
No idea what that piece of code is. My IDE apparently added it from nowhere.

Reply:
Reverted in 32bff7a1f97072c44eeec9b3753fe689566dbc70

Reply:
and in 4051b695b4298b6d4d82d2d091437353454bc9e7 added back newline